logstash-core/lib/logstash/pipeline.rb

```diff
@@ -106,6 +88,22 @@ def compile_lir
     ]
     LogStash::Compiler.compile_sources(sources_with_metadata, @settings)
   end
+   
+  def build_output(name, line, column, *args) 
```

Comment:
Are these to satisfy the `RubyIntegeration.Pipeline` interface? If so, can we align then names (use camelCase instead of the snake_case provided by JRuby?).

Rationale: grepping fails for 'buildOutput' to see where it is defined, so it was awkward to read the see the Pipeline interface and try to find an implementation.

Reply:
@jordansissel right makes sense :) => will adjust